src/api/image/bin/unsplash-download.js

```diff
@@ -16,8 +16,9 @@ const pipeline = promisify(stream.pipeline);
  * @param {string} url - the URL to the image
  * @param {string} filename - the local filename to use when writing
  */
-function downloadPhoto(url, filename) {
-  return pipeline(fetch(url), fs.createWriteStream(filename)).then(() =>
+async function downloadPhoto(url, filename) {
+  const response = await fetch(url);
```

Comment:
I think `await` is incorrect here.  Node has a number of programming models: synchronous, asynchronous, and streams.  In this code, you are trying to create a pipeline out of two streams, where the `fetch` stream gets piped into the file write stream.  That way, as the bytes flow in from the `fetch`, they will get written to disk without having to accumulate in memory.